manticore/models/system.py

```diff
@@ -0,0 +1,22 @@
+class OSException(Exception):
```

Comment:
This system.py is a desperate attempt to abstract the operating system models. Which think its now .. platform. So in the case that exception is actually used it should go somewhere in platform/. tbc..